### PR TITLE
Update Pre-Release

### LINKED ISSUE #24: Add Classifiers and URLs before release
pyproject.toml has some TODOs before release.

### PR DESCRIPTION
**Pull request by pauladkisson**
Fixes #24

This PR adds updates needed before release of v0.1.0 including various github workflows and updating the pyproject.toml.

Some github secrets have been added.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -35,15 +32,14 @@ keywords = [
     'ndx-extension',
 ]
 dependencies = [
-    "pynwb>=2.6.0",
-    "hdmf>=3.13.0",
+    "pynwb>=2.8.1",
+    "hdmf>=3.14.2",
 ]
```
Comment (luiztauffer) [inline pyproject.toml:37]: I can confirm this works with `pynwb==2.8.0` and `hdmf==3.14.1`
maybe we pin it to the lowest minor versions? @pauladkisson

**Comment by luiztauffer** [comment]
@pauladkisson what is this workflow needed for? `assess-file-changes.yml`

**Comment by pauladkisson** [comment]
> what is this workflow needed for? `assess-file-changes.yml`

`assess-file-changes.yml` is used by `detect-changelog-updates.yaml` to automatically detect if the changelog needs to be updated.

File: requirements-min.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # minimum versions of package dependencies for installation
 # these should match the minimum versions specified in pyproject.toml
 # NOTE: it may be possible to relax these minimum requirements
-pynwb==2.8.1
-hdmf==3.14.2
+pynwb==2.8.0
+hdmf==3.14.1
```
Comment (luiztauffer) [inline requirements-min.txt:5]:
```suggestion
pynwb>=2.8.0
hdmf>=3.14.1
```

**Pull request merged**